staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -103,12 +103,23 @@ var (
 			// This metric is used for verifying api call latencies SLO,
 			// as well as tracking regressions in this aspects.
 			// Thus we customize buckets significantly, to empower both usecases.
-			Buckets: []float64{0.05, 0.1, 0.15, 0.2, 0.25, 0.3, 0.35, 0.4, 0.45, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0,
-				1.25, 1.5, 1.75, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 40, 50, 60},
+			Buckets:        []float64{0.05, 0.1, 0.2, 0.4, 0.6, 0.8, 1.0, 1.25, 1.5, 2, 3, 4, 5, 6, 8, 10, 15, 20, 30, 45, 60},
```

Comment:
Could we perhaps move that change to a separate PR since it is somehow unrelated? We could also close the following issue from the new PR: https://github.com/kubernetes/kubernetes/issues/105346.

Also, we might want to mention this change in the release note since it can break existing consumers of the `apiserver_request_duration_seconds` such as kubernetes-mixins before https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/687. As Han mentioned in https://github.com/kubernetes/kubernetes/pull/105890#discussion_r737698313, it is allowed to modify label values and buckets since it will not impact ingestion, but we should still notify the users of such change to not break their setups.

Reply:
nit: please move the indentation fix to the other commit

Regarding the comment - I don't have strong opinion, but if SIG instrumentation wants that in a separate PR, let's do that.

Reply:
Created new pr: https://github.com/kubernetes/kubernetes/pull/106306

About indentation, since value is in 1 line I think the indents needs to be like that, or verify-gofmt.sh fails. I'll break it into 2 lines.